app/src/main/java/ingsw/group1/findmyphone/fragment/LogFragment.java

```diff
@@ -0,0 +1,69 @@
+package ingsw.group1.findmyphone.fragment;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import ingsw.group1.findmyphone.R;
+import ingsw.group1.findmyphone.event.SMSLogDatabase;
+import ingsw.group1.findmyphone.log.LogItemFormatter;
+import ingsw.group1.findmyphone.log.LogManager;
+import ingsw.group1.findmyphone.log.LogRecyclerAdapter;
+
+/**
+ * Fragment holding the Log screen. Contains a list of the Log events.
+ *
+ * @author Riccardo De Zen.
+ */
+public class LogFragment extends Fragment {
+
+    private SMSLogDatabase logDatabase;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'logDatabase' could be replaced by a local variable.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5010214)